build/release/teamcity-make-and-publish-build.sh

```diff
@@ -9,8 +9,8 @@ export BUILDER_HIDE_GOPATH_SRC=1
 
 build/builder.sh make .buildinfo/tag
 build_name="${TAG_NAME:-$(cat .buildinfo/tag)}"
-release_branch="$(echo "$build_name" | grep -Eo "^v[0-9]+\.[0-9]+")"
-is_custom_build="$(echo "$TC_BUILD_BRANCH" | grep -Eo "^custombuild-")"
+release_branch="$(echo "$build_name" | grep -Eo "^v[0-9]+\.[0-9]+" || echo"")"
```

Comment:
Let's add a comment explaining the need for `echo""`?

Reply:
Added.